### PR TITLE
ScummVM 1.8.0, new recipes

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by mmuman** [comment]
Would you mind rebasing all this?

**Comment by Begasus** [comment]
We were in the middle of rebasing, didn't have a chance yet to go all the way, hope someone could guid me through :)

**Comment by mmuman** [comment]
Yes I looked up for you on IRC the whole week :P

File: games-engines/scummvm/scummvm-1.8.0.recipe [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+SECONDARY_ARCHITECTURES="x86"
+
+ADDITIONAL_FILES="scummvm.rdef"
+
+PROVIDES="
+	scummvm$secondaryArchSuffix = $portVersion
+	app:scummvm$secondaryArchSuffix = $portVersion
+"
+
+REQUIRES="
```
Comment (KapiX) [inline games-engines/scummvm/scummvm-1.8.0.recipe]: You are missing `lib:libbz2`, `lib:libgl` and `lib:libglu`.

**Pull request merged**